test/SerialIOTest.cpp

```diff
@@ -91,8 +91,95 @@ TEST_CASE( "multi_series_test", "[serial]" )
     allSeries.clear();
 }
 
-inline
 void
+close_iteration_test( std::string file_ending )
+{
+#if openPMD_HAVE_ADIOS1 && !openPMD_HAVE_ADIOS2
+    bool isAdios1 = file_ending == "bp";
+#elif openPMD_HAVE_ADIOS1 && openPMD_HAVE_ADIOS2
+    bool isAdios1 = file_ending == "bp" &&
+        auxiliary::getEnvString("OPENPMD_BP_BACKEND", "NOT_SET") == "ADIOS1";
+#else
+    bool isAdios1 = false;
+#endif
+    std::string name = "../samples/close_iterations_%T." + file_ending;
+
+    std::vector<int> data{2, 4, 6, 8};
+    // { // we do *not* need these parentheses
+    Series write(name, Access::CREATE);
+    {
+        Iteration it0 = write.iterations[ 0 ];
+        auto E_x = it0.meshes[ "E" ][ "x" ];
+        E_x.resetDataset( { Datatype::INT, { 2, 2 } } );
+        E_x.storeChunk( data, { 0, 0 }, { 2, 2 } );
+        it0.close( /* flush = */ false );
+    }
+    write.flush();
+    // }
+
+    if (isAdios1)
```

Comment:
There is a new `Series::backend()` aka `AbstractIOHandler::backendName()` that you can query instead.